app/navigation/nonsipp/SharesNavigator.scala

```diff
@@ -161,7 +161,21 @@ object SharesNavigator extends JourneyNavigator {
       controllers.nonsipp.shares.routes.SharesIndependentValuationController.onPageLoad(srn, index, NormalMode)
 
     case SharesIndependentValuationPage(srn, index) =>
+      userAnswers.get(WhyDoesSchemeHoldSharesPage(srn, index)) match {
+        case Some(Acquisition) =>
+          controllers.routes.UnauthorisedController.onPageLoad()
+        case _ =>
```

Comment:
If it's a `Contribution` or `Transfer`, then I think the navigation should go to `SharesTotalIncomeController` in any case, regardless of whether `TypeOfShares` is `SponsoringEmployer`, `Unquoted`, or `ConnectedParty`.

Reply:
we are discussing same here, changing now